<launch>
    <node name="cone_connection_node" pkg="rr_evgp" type="cone_connection_node" output="screen">
        <param name="cones_topic" value="/lidar_up"/>
        <param name="walls_topic" value="/global_walls_topic"/>
        <param name="distance_between_walls" value="18"/>
        <param name="distance_between_cones" value="6"/>
        <param name="cone_connection_status" value="/cone_connection_status"/>
    </node>
</launch>
